client/my-sites/customer-home/main.jsx

```diff
@@ -437,10 +547,14 @@ const connectHome = connect(
 			isChecklistComplete: isSiteChecklistComplete( state, siteId ),
 			isAtomic: isAtomicSite( state, siteId ),
 			isStaticHomePage: 'page' === getSiteOption( state, siteId, 'show_on_front' ),
+			siteHasPaidPlan: isSiteOnPaidPlan( state, siteId ),
+			isNewlyCreatedSite: isNewSite( state, siteId ),
 			siteIsUnlaunched: isUnlaunchedSite( state, siteId ),
 			staticHomePageId: getSiteFrontPage( state, siteId ),
 			showCustomizer: ! isSiteUsingFullSiteEditing( state, siteId ),
 			hasCustomDomain: getGSuiteSupportedDomains( domains ).length > 0,
+			user: getCurrentUser( state ),
+			...themeInfo,
```

Comment:
Curious why we don't just include `currentTheme` & `currentThemeId` with default / empty values?  Does it break something?

Reply:
Good question! I pulled it from the existing logic, but I haven't been able to test this use case by purchasing a theme since I can't do so through our signup flow.

Reply:
Aha, apparently we have a themes-in-signup flow where this gets taken into account: https://github.com/Automattic/wp-calypso/commit/e89f47cb793e8916f10b2a093cd35e118fea3693

I'll investigate more.